src/Core/src/Platform/Android/MauiScrollView.cs

```diff
@@ -291,7 +291,7 @@ void SmoothScrollTo(int x, int y, Action finished)
 			animator.Start();
 		}
 
-		void IOnScrollChangeListener.OnScrollChange(NestedScrollView v, int scrollX, int scrollY, int oldScrollX, int oldScrollY)
+		static void IOnScrollChangeListener.OnScrollChange(NestedScrollView v, int scrollX, int scrollY, int oldScrollX, int oldScrollY)
```

Comment:
Yep, it came back! 😅

Reply:
hum, why?

Reply:
That is a good question. @hartez do you maybe have any insights here? It looks like the VS CA sees that it shouldn't do this but dotnet format does add it. For these cases should we add some compiler directives to ignore it?